includes/dart-integrations/app-start-instrumentation.mdx

```diff
@@ -31,7 +31,7 @@ When the SDK receives the start and end times of the app launch, the SDK:
 
 <Alert>
 
-While the SDK differentiates between cold and warm starts, it doesn't track hot starts/resumes.
+Sentry's App Start instrumentation aims to be as comprehensive and representative of the user experience as possible, and adheres to guidelines by the platform vendors. For this reason, App Starts reported by Sentry might be longer than what you see in other tools. Reaad more on the [Apple App Start docs](/platforms/apple/guides/ios/tracing/instrumentation/automatic-instrumentation/#app-start-tracing) and [Android App Start docs](/platforms/android/tracing/instrumentation/automatic-instrumentation/#app-start-instrumentation).
```

Comment:
The apple and android app starts aren't really related to Flutter, should we keep the reference to them?

Reply:
ah thanks, that's a copy/paste error, I'll remove it